Documentation/ApiOverview/Backend/JavaScript/Modules/MultiStepWizard.rst

```diff
@@ -1,40 +1,144 @@
+..  _modules-multistepwizard:
 
 =================
 Multi-step wizard
 =================
 
-The JavaScript module :js:`MultiStepWizard` can be used to show a modal multi-step
-wizard with the following features:
-
-* Navigation to previous / next steps
-* Steps may have descriptive labels like "Start" or "Finish!"
-* Steps may require actions before becoming available.
-
-Code examples:
-
-.. code-block:: js
-
-   // Show/ hide the wizard
-   MultiStepWizard.show();
-   MultiStepWizard.dismiss();
-
-   // Add a slide to the wizard
-   MultiStepWizard.addSlide(
-       identifier,
-       stepTitle,
-       content,
-       severity,
-       progressBarTitle,
-       function() {
-       ...
-       }
-   );
-
-   // Lock/ unlock navigation buttons
-   MultiStepWizard.lockNextStep();
-   MultiStepWizard.unlockNextStep();
-   MultiStepWizard.lockPrevStep();
-   MultiStepWizard.unlockPrevStep();
+The JavaScript module :js:`MultiStepWizard` can be used to show a modal
+multi-step wizard with the following features:
+
+*   Navigation to previous / next steps
+*   Steps may have descriptive labels like "Start" or "Finish!"
+*   Steps may require actions before becoming available.
+
+..  _modules-multistepwizard-addslide:
+
+Add Slide
+=========
+
+You have to define at least one slide :javascript:`MultiStepWizard.addSlide()`.
+
+..  confval-menu::
+    :name: multi-step-wizard-settings
+    :display: table
+    :type:
+
+    ..  confval:: identifier
+        :name: multi-step-wizard-settings-identifier
+        :Required: true
+        :type: string
+
+        A unique identifier for the slide
+
+    ..  confval:: title
+        :name: multi-step-wizard-settings-title
+        :Required: true
+        :type: string
+
+        The title of the slide. Will be shown as header of the slide.
+
+    ..  confval:: content
+        :name: multi-step-wizard-settings-content
+        :Required: true
+        :type: string|JQuery|Element|DocumentFragment
+
+        The content of the slide. If `string` any HTML will be escaped. To
+        prevent that wrap your content into a jquery object:
+
+        ..  code-block:: js
+
+            $(`Your HTML content`);
+
+    ..  confval:: severity
+        :name: multi-step-wizard-settings-severity
+        :Required: true
+        :type: SeverityEnum
+
+        Set severity color for sheet. Color will only affect title bar and
+        prev- and next-buttons.
+
+    ..  confval:: progressBarTitle
+        :name: multi-step-wizard-settings-progressBarTitle
+        :Required: true
+        :type: string
+
+        Set a title for the progress bar. The progress bar will only be shown
+        below the content section of the slide, if you have defined at least
+        two or more slides.
+
+    ..  confval:: callback
+        :name: multi-step-wizard-settings-callback
+        :Required: true
+        :type: SlideCallback
+
+        A JS callback function which will be called after the slide was
```

Comment:
```suggestion
        A JavaScript callback function which will be called after the slide was
```
Try to avoid abbreviations, they make reading harder for unexperienced users